array-manipulation-1/src/tail.js

```diff
@@ -0,0 +1,15 @@
+/* exported tail */
+
+/* declare a function tail with one parameter array
+declare a variable tail and assign it the value array literal
+loop through the array starting at one
+and assigning it to the tail array at iteration - 1
+return tail array  */
+
+function tail(array) {
+  var tail = [];
+  for (var i = 1; i < array.length; i++) {
+    tail[i - 1] = array[i];
```

Comment:
You don't need `tail[i-1]` here! You can just do `tail.push(array[i])`

Here's the full instructors solution!
```
function tail(array) {
  var output = [];
  for (var i = 1; i < array.length; i++) {
    var element = array[i];
    output.push(element);
  }
  return output;
}
```